bin/run-jest.js

```diff
@@ -16,4 +16,10 @@ const {
     runJest
 } = require('../utils/runJest');
 
-runJest();
\ No newline at end of file
+runJest().then((output) => {
+    if (output.results.numFailedTests > 0) {
```

Comment:
Should be `numFailedTestSuites`, so that a test suite that doesn't pass tslint also triggers a failure